### DISCUSSION TITLE
Credit Given in Changelog for v0.4.0

### DISCUSSION THREAD

**Discussion by sneakers-the-rat** [wehr-lab/autopilot#116] [Q&A]
Writing the changelog for v0.4.0 right now and want to make sure I credit anyone who has contributed in any way! If you don't see something you helped with listed here, let me know and i'll add you!

https://docs.auto-pi-lot.com/en/dev/changelog/v0.4.0.html

**Comment by roaldarbol** [comment]
Hi Jonny, just wanted to say thanks for this! Seeing myself in the changelog means a lot to me, made me quite proud in fact (despite not having helped much with the development) - it's nice to know that the time spent testing out and providing feedback actually helps, and is appreciated. I hope to return and spend more time with Autopilot once my current set of experiments are done and dusted.
So yeah, thanks again and keep up the awesome work! 
PS The wiki is incredible, it's clear a LOT of thought and research has gone into designing it!

**Reply by sneakers-the-rat** [reply]
Of course! Very much appreciate you saying so and your involvement in the project, taking a risk on young software is not something I take lightly! I want to cultivate a norm of credit assignment and mutual development, move away from this package just being something *i* make to something *we* make <3

**Reply by sneakers-the-rat** [reply]
good luck on the current experiments!!!!!!!

**Reply by roaldarbol** [reply]
_This_ is exactly why I love young open source projects. And thanks!
